pkg/controller/prometheusmetrics/prometheus.go

```diff
@@ -0,0 +1,66 @@
+package prometheusmetrics
```

Comment:
nit: would call this just "metrics"... not sure we are even going to have any other metrics